xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/client/transactions/TransactionResultJsonTests.java

```diff
@@ -0,0 +1,32 @@
+package org.xrpl.xrpl4j.model.client.transactions;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.junit.Test;
+import org.xrpl.xrpl4j.model.AbstractJsonTest;
+import org.xrpl.xrpl4j.model.transactions.Hash256;
+import org.xrpl.xrpl4j.model.transactions.ImmutablePayment;
+
+public class TransactionResultJsonTests extends AbstractJsonTest {
+
+  @Test
+  public void deserializePaymentTransactionResult() throws JsonProcessingException {
```

Comment:
Can we add test coverage around serialize -> deserialize? Or is that perhaps covered elsewhere?

Reply:
Originally, we decided this class and `TransactionResult` didn't need to be serialized, since they are just response objects and we'd need a custom serializer to serialize correctly.  The only reason I can think of to serialize would be if you were writing your own rippled in java.

Now that I've slept on it though, I think i'll add that custom serializer, just so the response objects are consistent